### ISSUE TITLE
"Open this link in a new tab" not working

### ISSUE THREAD

**Issue by ZohaibHaider7** [markjaquith/page-links-to#171]
Hi Mark!
I am using this plugin (latest version) in WordPress version 4.9.8. The issue is that when I give a custom URL and check "Open this link in a new tab", it appends "#new_tab" at the end of the URL, but the link is not opening in a new tab. It opens in the same tab with the URL like this: "http://example-site.com#new_tab".

P.S. I am using a custom theme.
![Screenshot_8](https://github.com/markjaquith/page-links-to/assets/68470035/2220b5d3-f4a2-4974-ba3c-231d40fd1470)
![Screenshot_9](https://github.com/markjaquith/page-links-to/assets/68470035/1af2fa20-02a6-4215-b4a8-7e2390398f3e)
![Screenshot_10](https://github.com/markjaquith/page-links-to/assets/68470035/3d1f8070-ee82-4e42-b8ad-78b277900d54)

**Comment by jazdauskas** [comment]
Yes, I'm having the exact issue. I only had this issue after the latest update of WordPress. I'm running the 6.2.2 version.

**Comment by scacchetti** [comment]
Same issue, WP 6.2.2

**Comment by MadtownLems** [comment]
I ran into this problem, and in my debugging, found that the new-tab.js file was being blocked by my client. Disabling my ad blocker for the site domain seemed to resolve the issue. I'm curious if this fixes it for y'all, as well.

Of course, what would be great is figuring out WHY my adblocker was blocking it and prevent it in the future.

**Comment by hooker99** [comment]
> I ran into this problem, and in my debugging, found that the new-tab.js file was being blocked by my client. Disabling my ad blocker for the site domain seemed to resolve the issue. I'm curious if this fixes it for y'all, as well.
> 
> Of course, what would be great is figuring out WHY my adblocker was blocking it and prevent it in the future.

AdBlockers can block JavaScript (spam etc.). I don't know if it's possible, but plugin author should use HTML: <a target="_blank" href="url_here_or_link">Click me</a>

You can do a little test: open your blog/website and press the F12 (open console) on your blog and add target="_blank" to the link, then click this link in the browser. AdBlock won't block it.
